src/main/java/no/nav/pto/veilarbportefolje/elastic/ElasticIndexer.java

```diff
@@ -263,33 +265,44 @@ private void slettBrukereIkkeLengerUnderOppfolging(List<OppfolgingsBruker> bruke
     }
 
     @SneakyThrows
-    public Result<OppfolgingsBruker> slettBruker(OppfolgingsBruker bruker) {
-
-        List<Integer> backoffs = Arrays.asList(1000, 3000, 6000, 0);
-
-        for (Integer backoff : backoffs) {
+    public void slettBruker(OppfolgingsBruker bruker) {
+
+        if (unleashService.isEnabled(Feature.MARKER_SOM_SLETTET)) {
```

Comment:
maybe its better to move this code to seperate function, so slettBruker stays with reasonable size.

Reply:
I agree, fixed!